src/mito2/src/worker/handle_manifest.rs

```diff
@@ -329,6 +329,15 @@ async fn edit_region(
 
             let index_key = IndexKey::new(region_id, file_meta.file_id, FileType::Parquet);
             let remote_path = location::sst_file_path(layer.region_dir(), file_meta.file_id);
+
+            let is_index_exist = file_meta.exists_index();
+            let index_file_size = file_meta.index_file_size();
+
+            let index_file_index_key =
+                IndexKey::new(region_id, file_meta.file_id, FileType::Puffin);
+            let index_remote_path =
+                location::index_file_path(layer.region_dir(), file_meta.file_id);
+
             let file_size = file_meta.file_size;
             common_runtime::spawn_global(async move {
```

Comment:
Can we add a gauge for the number of inflight download tasks? So we can know the download progress.